### PR TITLE
Need to have perf, pstack, gdb and netstat installed on all vespa run…

### PR DESCRIPTION
**Pull request by baldersheim**
…times.
@aressem PR
@bjorncs FYI

### CODE REVIEW

**Comment by baldersheim** [comment]
@vekterli @geirst FYI

File: dist/vespa.spec [hunk 1]
```diff
@@ -96,8 +96,12 @@ Requires: lz4
 Requires: libzstd
 Requires: zlib
 Requires: libicu
+Requires: perf
+Requires: gdb
```
Comment (aressem) [inline dist/vespa.spec]: Do you need the standard gdb, when you get the devtoolset-7-gdb installed ?

Comment (baldersheim) [reply dist/vespa.spec]: That was to cover non-centos.

Comment (aressem) [reply dist/vespa.spec]: But it is in the common section so you get the regular gdb installed as well on CentOS. Better to put this under the Fedora part.

Also these are packages that Vespa does not require to fully function and should really be added elsewhere for convenience.

Comment (baldersheim) [reply dist/vespa.spec]: We can live fine with default gdb. The important one is pstack.

I agree that in a pure world you can live without these. But any system without perf, pstack, and netstat are more of theoretical interest. I see that the vespa-perftop and so is not part of vespa, but vespa-yahoo. That I think should change.

**Pull request merged**